src/pages/graphile-build/plugin-options.md

```diff
@@ -6,9 +6,8 @@ title: Plugin Options
 
 ## Plugin Options
 
-The second argument to `buildSchema` is [the
-options](/graphile-build/plugin-options/) which are made available to every
-plugin (as their second argument).
+The second argument to the [`graphile-build` functions](http://localhost:8000/graphile-build/graphile-build/#the-graphile-build-module)
+is an **`options`** object with arbitrary propreties which is made available to every plugin as its second argument.
```

Comment:
```suggestion
is an **`options`** object with arbitrary properties which is made available to every plugin as its second argument.
```

I think it's also included on the `build` object now? I have a feeling that this turned out to be a better approach, but I don't remember the details off-hand.